data/dineshlalam15.json

```
@@ -13,5 +13,15 @@
       "url": "https://github.com/dineshlalam15",
       "icon": "FaGithub"
     }
+    {
+      "name": "Hey! Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/dinesh-kumar-lalam-2584a8207/",
+      "icon": "FaLinkedIn"
+    }
```

Please add a missing comma `,` here.

```suggestion
    },
```